source/frontend/src/components/Profile/index.tsx

```diff
@@ -170,6 +228,41 @@ const Profile = (props: Props) => {
             </div>
           </div>
         </div>
+        {username && (
```

Comment:
The button should not be transparent, color should be given -referably different colors for different states-.

Reply:
![image](https://user-images.githubusercontent.com/44745432/147934317-4cad2f31-1c72-410c-9f7b-951c2be764bc.png)
![image](https://user-images.githubusercontent.com/44745432/147934347-eee9ea3c-c1d7-4025-aa02-4aad9f5ff8da.png)